src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -1420,22 +1499,30 @@ Debug("GenerateVoidILStartupMethod: Linux: Setting the PInvoke native profiler l
 
   hr = rewriter_void.Export();
   if (FAILED(hr)) {
-    Warn("GenerateVoidILStartupMethod: Unable to save the IL body. ModuleID=", module_id);
+    Warn("GenerateVoidILStartupMethod: Unable to save the IL body. ModuleID=",
+         module_id);
     return hr;
   }
 
   return S_OK;
 }
 
 #ifndef _WIN32
-extern uint8_t dll_start[] asm("_binary_Datadog_Trace_ClrProfiler_Managed_Loader_dll_start");
-extern uint8_t dll_end[] asm("_binary_Datadog_Trace_ClrProfiler_Managed_Loader_dll_end");
-
-extern uint8_t pdb_start[] asm("_binary_Datadog_Trace_ClrProfiler_Managed_Loader_pdb_start");
-extern uint8_t pdb_end[] asm("_binary_Datadog_Trace_ClrProfiler_Managed_Loader_pdb_end");
+extern uint8_t dll_start[] asm(
+    "_binary_Datadog_Trace_ClrProfiler_Managed_Loader_dll_start");
+extern uint8_t dll_end[] asm(
+    "_binary_Datadog_Trace_ClrProfiler_Managed_Loader_dll_end");
+
+extern uint8_t pdb_start[] asm(
+    "_binary_Datadog_Trace_ClrProfiler_Managed_Loader_pdb_start");
+extern uint8_t pdb_end[] asm(
+    "_binary_Datadog_Trace_ClrProfiler_Managed_Loader_pdb_end");
 #endif
 
-void CorProfiler::GetAssemblyAndSymbolsBytes(BYTE** pAssemblyArray, int* assemblySize, BYTE** pSymbolsArray, int* symbolsSize) const {
+void CorProfiler::GetAssemblyAndSymbolsBytes(BYTE** pAssemblyArray,
```

Comment:
Would you be able to undo the formatting changes in this file to make it easier to identify the meaningful diff?